census/src/test/scala/model/RssdToMsaSpec.scala

```diff
@@ -0,0 +1,13 @@
+package model
+
+import hmda.census.model.RssdToMsa
+import org.scalatest.{ MustMatchers, WordSpec }
+
+class RssdToMsaSpec extends WordSpec with MustMatchers {
+  "RssdToMsa" must {
+    "map values correctly" in {
+      val map = RssdToMsa.map
+      map.keys.toList.length mustBe 123703
```

Comment:
No, because each line in the file is a branch RSSD, with a parent RSSD.  We ignore the branch RSSD and only map the parent RSSD to the MSA/MD.  So, to put it simply, the line count of the `branch_msa.csv` file is the number of branches we're counting, not the number of institutions.